.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "mixpanel",
+  "globals": {
+    "require": true,
```

Comment:
Since we only use `require` for those few scripts in the devtools dir, I think it makes more sense to declare this in those files directly, to help enforce `import` consistency throughout the rest of the codebase.

Reply:
Fixed. 

Although I haven't fixed all the linting issues in the rest of code base as part of this PR. Didn't want the code changes in this PR to balloon out because of eslint issues. I just wanted to see the squiggles when editing the publish script.

I'll send you another PR for that.

Reply:
Yeah, I've been holding off on standardizing formatting according to linter until after https://github.com/mixpanel/panel/pull/18 lands, just to avoid excessive merge cleanup...